### PR TITLE
fix(makefile): fix missing separator error

### PR DESCRIPTION
**Pull request by arschles**
cc/ @smothiki @technosophos

### CODE REVIEW

**Comment by arschles** [comment]
@smothiki can you review please

File: Makefile [hunk 1]
```diff
@@ -7,7 +7,7 @@ export GO15VENDOREXPERIMENT=1
 VERSION := 0.0.1-$(shell date "+%Y%m%d%H%M%S")
 LDFLAGS := "-s -X main.version=${VERSION}"
 BINDIR := ./rootfs/bin
-DEV_REGISTRY ?= $(eval docker-machine ip deis):5000
+DEV_REGISTRY ?= $(docker-machine ip deis):5000
```
Comment (technosophos) [inline Makefile:10]: Shouldn't this be `$(shell docker-machine ip deis)`

Comment (arschles) [inline Makefile:10]: don't think so, don't expect this output would need expansion. also discussed [here](https://github.com/deis/minio/pull/3#discussion_r45889527)

**Comment by arschles** [comment]
merging, we can add in the `shell` expansion later if we decide

**Pull request merged**